CI/env.sh

```diff
@@ -2,47 +2,26 @@
 
 export jobs=`grep -c processor /proc/cpuinfo`
 
-export geant4_version=10.05
-export geant4_shasum=4b05b4f7d50945459f8dbe287333b9efb772bd23d50920630d5454ec570b782d
-
-export build_dir=/root/build/${COMPILER}
-export install_dir=/root/opt/${COMPILER}
-
 export CTEST_OUTPUT_ON_FAILURE=1
 
-if [ "${HDF5_VERSION}" == "system" ]; then
-  export hdf5_build_dir=
-  export hdf5_install_dir=/usr/lib/x86_64-linux-gnu/hdf5/serial
-else
-  export hdf5_build_dir=${build_dir}/hdf5-${HDF5_VERSION}
-  export hdf5_install_dir=${install_dir}/hdf5-${HDF5_VERSION}
-fi
-
-export hdf5_1_10_4_shasum=8f60dc4dd6ab5fcd23c750d1dc5bca3d0453bdce5c8cdaf0a4a61a9d1122adb2
-
-
-export geant4_build_dir=${build_dir}/geant4-${geant4_version}
-export geant4_install_dir=${install_dir}/geant4-${geant4_version}
-
-export moab_build_dir=${build_dir}/moab-${MOAB_VERSION}-hdf5-${HDF5_VERSION}
-export moab_install_dir=${install_dir}/moab-${MOAB_VERSION}-hdf5-${HDF5_VERSION}
-
-export dagmc_build_dir=${build_dir}/DAGMC-moab-${MOAB_VERSION}-hdf5-${HDF5_VERSION}
-export dagmc_install_dir=${install_dir}/DAGMC-moab-${MOAB_VERSION}-hdf5-${HDF5_VERSION}
-
+export dagmc_build_dir=${build_dir}/DAGMC
+export dagmc_install_dir=${install_dir}/DAGMC
 export dagmc_build_dir_shared=${dagmc_build_dir}-shared
 export dagmc_install_dir_shared=${dagmc_install_dir}-shared
 
 export dagmc_build_dir_static=${dagmc_build_dir}-static
 export dagmc_install_dir_static=${dagmc_install_dir}-static
 
 
+# Compiler related variables
+
+export  FC=`which gfortran`
```

Comment:
Do we actually use this anywhere in CI?

Reply:
I don't know.
I could try to remove it....

Reply:
it is used in our various `build_XXXX.sh` scripts

Reply:
Perhaps it is also used elsewhere in the scripts - I only searched the PR as shown by GH.  However, I don't think we build any fortran in CI or docker???

Reply:
I don't think either.

I just put it there, because it looked better, without thinking more about it...

Reply:
we do install `gfortran` in docker

Reply:
I think the only place we use FORTRAN is for building Fluka and MCNP.  We don't do either of those in CI (right now).  That said, it might be wise to keep it around in case developers want to use our images as part of their own workflow for building/testing.